infrastructure-scripts/claude_start.sh

```
@@ -0,0 +1,1640 @@
+
+# Removed: Self-hosted LLM helper functions
+# These are now handled by the dedicated claude_llm_proxy repository
+# See: https://github.com/$USERorg/claude_llm_proxy
+
+# SSH tunnel PID file
+SSH_TUNNEL_PID_FILE="/tmp/qwen_ssh_tunnel.pid"
+
+# Cleanup function for SSH tunnels
+cleanup_ssh_tunnel() {
+    if [ -f "$SSH_TUNNEL_PID_FILE" ]; then
+        local PID=$(cat "$SSH_TUNNEL_PID_FILE")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)